07. 정렬과 그리디, 결정알고리즘(이분검색)/09. 결혼식/이서인/index.js

```diff
@@ -1,5 +1,22 @@
 function solution(times) {
-  let answer = Number.MIN_SAFE_INTEGER;
+  let answer = 0;
+  let preEnd = 0;
+  let currentPeople = 0;
+
+  const sortedTimes = times.sort((a, b) => {
+    a[0] - b[0];
+  });
```

Comment:
콜백 함수에서 `return` 문을 작성하지 않고 있어서 의도하신 `sortedTimes`는 나오지 않고 있을 겁니다!
인수의 순서만 바꿨을 때 출력 결과가 `2`에서 `3`으로 달라지게 되네요.

```js
let arr = [[12, 15], [14, 18], [15, 20], [20, 30], [5, 14]];
```

```suggestion
  const sortedTimes = times.sort((a, b) => a[0] - b[0]);
```

그리고 올바른 정렬로 변경했을 경우는 출력 결과가 `4`로 또 다르게 나오고 있어요!

Reply:
조건식 수정했습니다! 예리하시군요

```suggestion
    if (start >= preEnd - 1) {
```